### ISSUE TITLE
Avoid sun.misc.BASE64Encoder

### ISSUE THREAD

**Issue by raboof** [whitesource/agents#13]
As that no longer exists when running jdk9.

The 'official' replacement is `java.util.Base64`, but that is only available from jdk8 onwards, and https://github.com/whitesource/agents/commit/fe086f46527d5712e36640ffab74348d41d3e767 suggests you still support java 7

**Comment by dwijnand** [comment]
I use http://www.jenv.be and I solved this by running like this:

```bash
JAVA_HOME=$(JENV_VERSION=1.7 jenv javahome) mvn whitesource:update
```

**Comment by raboof** [comment]
That's a valid workaround, but does require that you have jenv and jdk1.7 installed. It would be nice not to have to require that.

**Comment by raboof** [comment]
This was already fixed in a1e063b3 (though the dependency on commons-codec might no longer be necessary since whitesource-agents is on java8 since 555d4d08 so java.util.Base64 is available now)
